Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/CGRect+Extensions.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Geometry+Extensions.swift
+//  MealTrackingPilot
+//
+//  Created by sugapriya on 09/09/19.
+//  Copyright © 2019 Intrepid. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public extension CGRect {
+    
+     var topLeft: CGPoint { return CGPoint(x: minX, y: minY) }
```

Comment:
Please change below properties in correct format like

var topLeft: CGPoint { 
     return CGPoint(x: minX, y: minY) 
}

Reply:
done changes